### PR TITLE
made adjustments for new datagrid

### PR DESCRIPTION
**Pull request by michaelzangerle**
**Tasks:**
- [x] done
- [ ] got feedback

**Informations:**

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| Tests pass? | none |
| Fixed tickets | none |
| Doc | none |

### CODE REVIEW

File: Resources/public/js/components/shippings/components/list/main.js [hunk 1]
```diff
@@ -12,41 +12,47 @@ define(['sulusalesshipping/util/sidebar'], function(Sidebar) {
...
     var bindCustomEvents = function() {
-        // navigate to edit contact
-        this.sandbox.on('husky.datagrid.item.click', function(id) {
-            // get data for sidebar via controller
-            this.sandbox.emit('salesorder.orders.sidebar.getData', {
-                data: id,
-                callback: function(contact, account) {
-                    this.sandbox.emit(
-                        'sulu.sidebar.set-widget',
+            // navigate to edit contact
```
Comment (christianbader) [inline Resources/public/js/components/shippings/components/list/main.js:15]: indent

**Comment by christianbader** [comment]
tests? else ack

File: Resources/public/js/components/shippings/components/list/main.js (hunk 1 above)
Comment (michaelzangerle) [inline Resources/public/js/components/shippings/components/list/main.js:15]: this is because of the var keyword

**Comment by michaelzangerle** [comment]
tests have been broken by someone else - will be fixed in different PR

**Pull request merged**